abstractions/src/OpenSearch.Stack.ArtifactsApi/ServerType.cs

```diff
@@ -31,8 +31,6 @@ namespace OpenSearch.Stack.ArtifactsApi
 	public enum ServerType
 	{
 		OpenSearch,
-		OpenDistro,
-		ElasticSearch,
 		DEFAULT = OpenSearch
 	}
 }
```

Comment:
Perhaps, you can remove this entire enum and all references. It was added just to support testing on OpenDistro.

Reply:
I've completely removed the enum now